src/web/components/SharingPermission/ParticipantItem.tsx

```diff
@@ -51,7 +51,7 @@ export function ParticipantItem({ participant, onClick, checked, addedBy }: Part
           onClick={onClick}
           status={checked}
           className='participant-checkbox'
-          disabled={addedBy === 'Auto'} // addedBy is currently hardcoded to 'Manual'
+          disabled={!!addedBy && !addedBy.includes('Manually Added')}
```

Comment:
Doing a string comparison here seems like a code smell - I'm not sure about it. At least there should be a `const` somewhere but it makes me think there might be something else to look into.